client/src/components/DataFiles/DataFilesSystemSelector/DataFilesSystemSelector.js

```diff
@@ -74,7 +75,9 @@ const DataFilesSystemSelector = ({
         disabled={disabled}
       >
         {systemList
-          .filter(s => s.scheme !== 'projects')
+          .filter(
+            s => s.scheme !== 'projects' && !excludedSystems.includes(s.system)
```

Comment:
Would it be more straightforward to make the `api === 'tapis'` a requirement, rather than a exclusion list?

Reply:
No, there are cases where we want non-tapis systems in the list (copy) and systems where we don't (app selector).

Reply:
Ah right 👍